recipes/kraken/meta.yaml

```diff
@@ -23,11 +23,20 @@ build:
 requirements:
   build:
     - perl
-    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
 
+  host:
+    - perl
+    - clangdev  # [osx]
+    - llvm-openmp  # [osx]
   run:
-    - jellyfish 1.*
+    - bioconda::jellyfish 1.*
     - perl
+    - rsync
+    - wget
+    - libcxx  # [osx]
```

Comment:
Was this one necessary? It should not be...

Reply:
it did not find openmp without it. we noticed the same thing for [kraken2](https://github.com/bioconda/bioconda-recipes/pull/14971)

Reply:
mmmh, my bad, it seems to build without it.